packages/util-lang/lib/isInstanceInProtoChain.test.mjs

```diff
@@ -1,8 +1,56 @@
 import test from '@zorko-io/tool-test-harness'
 import {isInstanceInProtoChain} from './isInstanceInProtoChain'
 
-test('async - parses valid value', async (t) => {
-  // TODO: gh-82 fix existing test, provide few more use cases
+/*
+A2
+|
+A1 -> A3     B
+|
+A
+*/
+class A {}
+const a = new A()
 
-  t.assert(isInstanceInProtoChain({}, Object), true)
+class A1 extends A {}
+const a1 = new A1()
+
+class A2 extends A {}
+const a2 = new A2()
+
+class A3 extends A1 {}
+const a3 = new A3()
+
+class B {}
+const b = new B()
+
+test('should return true because a has A.prototype in his prototype chain', (t) => {
```

Comment:
In the current situation, it doesn't make sense to split the test case with `test` it's too noisy 

better to something like 

```
// just one 'test' for all
test('basic checks', t => {

  t.true(
    isInstanceInProtoChain(a1, A),
   'should return true because a1 has A.prototype in his prototype chain'
  )
// other checks here
})

```